script/prerender/contentPages.tsx

```diff
@@ -268,7 +268,7 @@ function injectHTML(html: string, {body, styles, state, fonts, meta, links, modu
     `<script>window.__PRELOADED_STATE__ = ${JSON.stringify(state).replace(/</g, '\\u003c')}</script>`
   );
 
-  html = html.replace(/(<script src="[^"]+\.chunk\.js"><\/script>)/, scripts.join('') + '$1');
+  html = html.replace(/(<script defer="defer" src="[^"]+\.chunk\.js"><\/script>)/, scripts.join('') + '$1');
```

Comment:
The htmlWebpackPluginInstance config in the craco config renders the `defer="defer"` but seems like that is standard or something, though the inconsistency of the boolean attributes here bothers me a little.